patterns/services-3-col.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
- * Title: Services - Three Columns
- * Slug: twentytwentyfive/services-three-columns
+ * Title: 3 columns
+ * Slug: twentytwentyfive/services-3-col
```

Comment:
Is this meant to be Services, 3 columns, like the Pricing, 3 columns?

Reply:
Yes. One moment.